ocs_ci/deployment/deployment.py

```diff
@@ -455,6 +456,22 @@ def do_deploy_ocs(self):
         else:
             logger.warning("OCS deployment will be skipped")
 
+    def do_deploy_mce(self):
+        """
+        Deploy Multicluster Engine
+
+        """
+        if config.DEPLOYMENT.get("deploy_mce"):
+            if config.ENV_DATA.get("skip_mce_check_if_present"):
+                check_mce_deployed = False
+                check_mce_ready = False
+            else:
+                check_mce_deployed = True
```

Comment:
not fully agree with this workaround:

if "skip_mce_check_if_present" -> check_mce_deployed == True
this will mislead in future